campari/simulation.py

```diff
@@ -252,31 +252,33 @@ def simulate_wcs(angle=None, x_shift=None, y_shift=None, roman_path=None, base_s
     """
     rotation_matrix = np.array([np.cos(angle), -np.sin(angle), np.sin(angle),
                                np.cos(angle)]).reshape(2, 2)
-    image = fits.open(roman_path + f"/images/simple_model/{band}/" +
-                      f"{base_pointing}/Roman_TDS_simple_model_{band}_{base_pointing}"
-                      + f"_{base_sca}.fits.gz")
+
+    img_collection = ImageCollection()
+    img_collection = img_collection.get_collection("ou2024")
```

Comment:
Should this "ou2024" be taken from a config file or parameter argument somewhere?